src/color-picker/src/ColorPicker.tsx

```diff
@@ -481,7 +500,7 @@ export default defineComponent({
     function renderPanel (): VNode {
       const { value: rgba } = rgbaRef
       const { value: displayedHue } = displayedHueRef
-      const { internalActions, modes } = props
+      const { internalActions, modes, control } = props
```

Comment:
```suggestion
      const { internalActions, modes, control } = props
```

可以把 internalActions 变成公开的

Reply:
把 control 的功能用 actions 实现

Reply:
可以的